test/topology_raft_disabled/test_raft_upgrade.py

```diff
@@ -176,49 +176,6 @@ async def test_raft_upgrade_basic(manager: ManagerClient, random_tables: RandomT
     assert(table.full_name in rs[0].description)
 
 
-@pytest.mark.asyncio
```

Comment:
I the document in the commit message public? Does it make sense to link to google docs at all?

Reply:
It's shared with ScyllaDB group. I think it doesn't hurt to add the link.

Reply:
It does not, but imagine the annoyance of someone theoretical who read the commit message but cannot read the doc.

Reply:
I also happen to think it's better to put a link to apublic document. It's easy to add the text of the document to the issue or to a github wiki. It's approved now, and will likely stay frozen forever.